app/helpers/book_helper.rb

```diff
@@ -1,5 +1,9 @@
 module BookHelper
-    def csv_names list
-        list.pluck(:name).join(',')
+    def csv_names(list)
+        list.pluck(:name).join(', ')
+    end
+
+    def human_boolean(boolean)
+        boolean ? 'Yes' : 'No'
     end
```

Comment:
please don't use namings like : int, long, boolean, string
replace it with `flag`. 
or even better: make it as a checkbox or a switch button in the view

Reply:
localization :)